packages/plugins/rudder_plugin_interface/lib/src/constants.dart

```diff
@@ -31,6 +31,8 @@ class Constants {
   // whether we should record screen views automatically
   static const bool RECORD_SCREEN_VIEWS = false;
 
+  static const String DATA_RESIDENCY_VALUE = "US";
```

Comment:
i this a default value? if yes rename to DEFAULT_DATA_RESIDENCY or something

Reply:
sure, will change it

Reply:
updated it